.github/workflows/container-image.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and Push Container Image
+
+on:
+  push:
+    branches:
+      - main
+      - buildanduploadimage
```

Comment:
probably want to remove references to other temp branches here besides `main`

Reply:
Done.